tests/README.md

```diff
@@ -0,0 +1,81 @@
+## Backend development
+
+The backend test suite can be run locally. 
+
+### Start the services
+
+We need all but the main `backend` service running.  Since multiple `backend` instances are allowed, we will simply start everything up
+
+    # ENSURE THE IMAGES ARE CLEAN
+    docker-compose down
+    # SETUP ALL IMAGES
+    docker-compose up --build
+
+### Install into virtual environment
+
+Treeherder, requires specific library versions that will likely interfere with with what you have installed. It is best to setup a virtual environment to contain the quirks it introduces.
+
+Be sure you are in the `treeherder` main directory
+
+    pip install virtualenv
+    virtualenv venv
+    source venv/bin/activate
+    venv/bin/pip install -r requirements/common.txt
+    venv/bin/pip install -r requirements/dev.txt
+    venv/bin/pip install -r requirements/docs.txt
+
+...or Windows...
+
+    pip install virtualenv
+    virtualenv venv
+    venv\Scripts\activate
+    python -m pip install -r requirements\common.txt
+    python -m pip install -r requirements\dev.txt
+    python -m pip install -r requirements\docs.txt  
```

Comment:
Having said what I mentioned on the previous steps you can adjust these steps in the same way (no need for `virtualenv` as a dependency, skip `python -m`, skip docs.txt).